hs_core/autocomplete_light_registry.py

```diff
@@ -10,15 +11,7 @@ def choices_for_request(self):
         return super(UserAutocomplete, self).choices_for_request()
 
     def choice_label(self, choice):
-        label = ""
-
-        if choice.first_name:
-            label += choice.first_name
-
-        if choice.last_name:
-            if choice.first_name:
-                label += " "
-            label += choice.last_name
+        label = " ".join([choice.first_name or "", choice.userprofile.middle_name or "", choice.last_name or ""])
```

Comment:
@hyi Take a look at this part I modified before merging.

Reply:
@Maurier Looks good. Thanks for taking care of this before merging. I have also updated park so that @horsburgh can test this auto-complete update on park as well.